src/utils/api-request.ts

```diff
@@ -274,7 +285,7 @@ function enhanceError(
   return error;
 }
 
-function settle(resolve, reject, response: LowLevelResponse) {
+function finalizeRequest(resolve, reject, response: LowLevelResponse) {
```

Comment:
Javadoc missing.

Reply:
Done. I'm only adding a description to these un-exported helper functions.